README.md

```diff
@@ -1 +1,201 @@
+ 
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="https://cdn-icons-png.flaticon.com/512/3150/3150115.png" alt="logo" width="120"  height="auto" />
+  <br/>
+   
+
+  <h1><b>Leaderboard 🏆🏆🏆</b></h1>
+
+</div>
+<br>
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<br>
+
+<!-- DEPLOYMENT -->
+### This project goes live [here](https://frempongdev.github.io/Leaderboard/dist/) 🚀 .
+If link does not work copy URL ( https://frempongdev.github.io/Leaderboard/dist/ )
+
+<br>
+
+<!-- DEPLOYMENT -->
+<!-- ### Have a look at the project's video presentation [here](https://www.loom.com/share/1f7de2b9345a4d3988c6327952899d30) 📽️ . -->
+
+<br>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+
+**Leaderboard** is a single page application project for takinf and displaying scores. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">Link to HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">Link to CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Javascript</summary>
+  <ul>
+    <li><a href="https://www.javascript.com/">Link to Javascript</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **This Project has a Readme.md file**
+- **This Project has a gitignore file**
+- **This projects follows all Microverse Linters**
+- **This project takes user inputs of names and scores**
+- **This project displays the input for name and scores in the recent scores box**
+- **This project saves the scores details inside the Browser**
+- **This project has syntax with ES6 format.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to copy the link of this Repository.
+
+
+
+```sh
+ https://github.com/frempongdev/Leaderboard.git
+```
+<br>
+
+### Setup
+
+Clone this repository to your desired directory using the command: 
+
+
+```sh
+  cd your-folder
+  git clone https://github.com/frempongdev/Leaderboard.git 
+```
+
+<br>
+
+
+### Usage
+
+Kindly go ahead and give this project a massage but do not forget to give a detailed description in your pull request.
```

Comment:
- [ ] Kindly change this line to include the commands that the user should use in order to install dependencies and run the server because it will document your repo and make it clear how to use it

Reply:
Can you please provide an example of such a command or point me to helpful material?

Reply:
I just did some research and added some commands. Thank You.

Reply:
well resolved! ✔️